src/entries/popup/pages/home/Activity/ActivityDetails.tsx

```diff
@@ -248,7 +248,7 @@ function NetworkData({ transaction: tx }: { transaction: RainbowTransaction }) {
           </Inline>
         }
       />
-      <FeeData transaction={tx} />
+      {tx.status != 'pending' && <FeeData transaction={tx} />}
```

Comment:
I'm working with "stuck" transactions and it's weird to see the fee data rows being in a perpetual skeleton state when it's not even fetching it, but happy to revert if intended